ext/phar/phar_internal.h

```diff
@@ -459,9 +459,6 @@ typedef struct _phar_entry_data {
 	zend_off_t                    position;
 	/* for copies of the phar fp, defines where 0 is */
 	zend_off_t                    zero;
-	uint32_t             for_write:1;
-	uint32_t             is_zip:1;
-	uint32_t             is_tar:1;
```

Comment:
I might be wrong, but I have some weird recollection of there being an issue with ext/zip and Phar that required some changes to be able to extract a Phar archive, could you git blame and go through the history as to why those were added in the first place?

Reply:
ACK, I must be misremembering something then.